spec/models/user_building_spec.rb

```diff
@@ -1,4 +1,13 @@
 require 'rails_helper'
 
 describe UserBuilding do
+	it { should have_many :units }
+	it 'is invalid without an address' do
+		user_building = build(:user_building, address: nil)
+		user_building.valid?
+		expect(user_building.errors[:address]).to include("can't be blank")
+	end
+	it 'has as many addresses as user_buildings' do
+		expect (UserBuilding.all.count == UserBuilding.all_addresses.count)
```

Comment:
The purpose of this test is actually to test the .all_addresses class method that is defined in the UserBuilding model. Maybe I should add a describe block to make this clear?